### PR TITLE
Chapter 1 Homework 3

### PR DESCRIPTION
**Pull request by EMonfort**
Added the code for chapter 1 homework 3

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Person 1:");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Make this simply 2 function calls.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (EMonfort) [reply Src/BootCamp.Chapter/Lesson3.cs]: > Make this simply 2 function calls.

Made the requested changes

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -7,26 +7,9 @@ namespace BootCamp.Chapter
...
-            name = PromptString("Enter your name: ");
-            surname = PromptString("Enter your surname: ");
-            age = PromptInt("Enter your age: ");
-            height = PromptFloat("Enter your height: ");
-            weight = PromptFloat("Enter your weight: ");            
-            Console.WriteLine($"{name} {surname} is {age} years old, his weight is {weight} kg and his height is {height} cm.");
-            bmi = CalculateBmi(weight, height);
-            Console.WriteLine($"Your BMI is: {bmi}");
+        {            
+            CallFunctions("Person 1");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:11]: Name better.

**Review by Almantask — APPROVED** [review]
Looks good 😄

**Pull request closed without merging**